packages/reqresp/src/utils/protocolId.ts

```diff
@@ -0,0 +1,43 @@
+import {Encoding, Protocol} from "../types.js";
+
+/**
+ * https://github.com/ethereum/consensus-specs/blob/v1.2.0/specs/phase0/p2p-interface.md#protocol-identification
+ */
+export function formatProtocolID(protocolPrefix: string, method: string, version: number, encoding: Encoding): string {
+  return `${protocolPrefix}/${method}/${version}/${encoding}`;
+}
+
+/**
+ * https://github.com/ethereum/consensus-specs/blob/v1.2.0/specs/phase0/p2p-interface.md#protocol-identification
+ */
+export function parseProtocolID(protocolId: string): Protocol {
```

Comment:
This function was not in source because it's only used in tests. I try to not keep any code in source that's only for testing